Makefile

```diff
@@ -59,36 +59,23 @@ clean:
 isclean:
 	(test "$(ALLOW_DIRTY_CHECKOUT)" != "false" || test 0 -eq $$(git status --porcelain | wc -l)) || (echo "Local git checkout is not clean, commit changes and try again." && exit 1)
 
-# One big sed command instead of a function because OPERATOR_X vars 
-# are provided by shell, not make vars, and hard (imposisble?) to
-# pass as args to a function.  
-SED_CMD=sed -e "s/\#IMAGE_REGISTRY\#/${IMAGE_REGISTRY}/g" \
-			-e "s/\#IMAGE_REPOSITORY\#/${IMAGE_REPOSITORY}/g" \
-			-e "s/\#IMAGE_NAME\#/${IMAGE_NAME}/g" \
-			-e "s/\#CATALOG_NAMESPACE\#/${CATALOG_NAMESPACE}/g" \
-			-e "s/\#CHANNEL\#/${CHANNEL}/g" \
-			-e "s/\#CATALOG_VERSION\#/${CATALOG_VERSION}/g" \
-			-e "s/\#CURRENT_COMMIT\#/${CURRENT_COMMIT}/g" \
-			-e "s/\#OPERATOR_NAME\#/$${OPERATOR_NAME}/g" \
-			-e "s/\#OPERATOR_NAMESPACE\#/$${OPERATOR_NAMESPACE}/g"
-
 .PHONY: manifests/catalog
-manifests/catalog:
+manifests/catalog: catalog
 	mkdir -p manifests/
 	# create CatalogSource yaml
 	TEMPLATE=scripts/templates/catalog.yaml; \
 	DEST=manifests/00-catalog.yaml; \
-	$(SED_CMD) $$TEMPLATE > $$DEST
+	$(call process_template,.,$$TEMPLATE,$$DEST)
```

Comment:
Since now we also have the `process_template` function in the `checkout-operator.mk` file, probably it would make sense to rename it to something with a broader meaning, like functions.mk, helpers.mk

Reply:
File is renamed.